src/main/java/seedu/weme/model/statistics/StatsManager.java

```diff
@@ -35,49 +36,41 @@ public StatsManager(Stats stats) {
     }
 
     //============= Like Data ====================================
-    /**
-     * Replaces the contents of the like data with {@code likeData}.
-     */
+
+    public void addDefaultLikeData(Meme meme) {
+        likeManager.initMemeLikeCount(meme);
+    }
+
     @Override
-    public void setLikeData(Map<String, Integer> likeData) {
+    public void setLikeData(Map<String, SimpleIntegerProperty> likeData) {
         likeManager.setLikeData(likeData);
     }
 
-    /**
-     * Returns the number of likes of a meme.
-     */
     @Override
     public int getLikesByMeme(Meme meme) {
         return likeManager.getLikesByMeme(meme);
     }
 
-    /**
-     * Returns an unmodifiable view of LikeData.
-     */
     @Override
-    public ObservableMap<String, Integer> getObservableLikeData() {
+    public ObservableMap<String, SimpleIntegerProperty> getObservableLikeData() {
         return likeManager.getObservableLikeData();
     }
 
-    /**
-     * Increments like count of a meme.
-     */
+    @Override
+    public void initMemeLikeCount(Meme meme) {
```

Comment:
remove this method and the related ones

Reply:
This method is actually in use.

Reply:
isn't it being used only by sampleWeme.initMemeLikeCount? addDefault does the same thing right